src/chia_dialect.rs

```diff
@@ -193,7 +202,10 @@ impl Dialect for ChiaDialect {
             // Extension 1 is for the keccak256 operator.
             1 => OperatorSet::Keccak,
 
-            // Extensions 2 and beyond are considered invalid by the mempool.
+            // Extension 2 is for the sha256tree operator.
+            2 => OperatorSet::Sha256tree,
+
+            // Extensions 3 and beyond are considered invalid by the mempool.
```

Comment:
This is also part of the `softfork` operator. I'm not sure we want to do this. I think it would keep things simpler by not adding another extension here.

Reply:
Reverted these changes now